packages/core/useFocus/index.ts

```diff
@@ -29,28 +24,27 @@ export interface FocusReturn {
  * Track or set the focus state of a DOM element.
  *
  * @see https://vueuse.org/useFocus
+ * @param target The target element for the focus and blur events.
  * @param options
  */
-export function useFocus(options: FocusOptions = {}): FocusReturn {
-  const {
-    initialValue = false,
-  } = options
+export function useFocus(target: MaybeElementRef, options: FocusOptions = {}): FocusReturn {
```

Comment:
Actually just had a thought, maybe we should rename `FocusOptions` to `UseFocusOptions` to also be consistent with our other APIs. Since this is already a breaking change we might as well.

Reply:
Ok, I renamed the return type to be `UseFocusReturn` as well.